overlord/snapstate/snapmgr.go

```diff
@@ -720,7 +727,7 @@ func (m *SnapManager) undoUnlinkCurrentSnap(t *state.Task, _ *tomb.Tomb) error {
 		return err
 	}
 
-	oldInfo, err := snapst.CurrentInfo(ss.Name)
+	oldInfo, err := snapst.CurrentInfo(ss.Name())
```

Comment:
now that we always set RealName, we could drop the name argument to SnapState.CurrentInfo, right?